src/ReactTinyLink.tsx

```diff
@@ -17,6 +17,8 @@ const fetchUrl = (
   defaultMedia: string,
   setData: (data: IReactTinyLinkData) => void,
   setLoading: (loading: boolean) => void,
+  onError: Function,
```

Comment:
Would it be great  if we can have `error` as a param like this ? 
`onError:(error: Error) => void,`